src/providers/ThemeProvider.tsx

```diff
@@ -1,14 +1,25 @@
 import { ThemeContext } from 'contexts/themeContext'
 import { useJuiceTheme } from 'hooks/JuiceTheme'
+import { useLayoutEffect } from 'react'
 
 export const ThemeProvider: React.FC = ({ children }) => {
   const juiceTheme = useJuiceTheme()
   const isDarkMode = juiceTheme.isDarkMode
 
+  // Set the theme on the body element
+  // This is needed for tailwind css dark theme classes to work
+  useLayoutEffect(() => {
```

Comment:
**suggestion/question**: we already do a similar thing here using the `color-scheme`: https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme

At the least we should colocate this logic (either in this file, or over in `JuiceTheme`), but perhaps we could actually consolidate it so tailwind users `color-scheme` property instead? not sure.

https://github.com/jbx-protocol/juice-interface/blob/b0f2ef0095d5a6905d46616ccc29d738605be65e/src/hooks/JuiceTheme.ts#L65-L71